@xen-orchestra/backups/_runners/_vmRunners/_AbstractXapi.mjs

```diff
@@ -257,6 +261,22 @@ export const AbstractXapi = class AbstractXapiVmBackupRunner extends Abstract {
         }
       })
     })
+
+    // now that we use CBT, we can destroy the data of the snapshot used for this backup
+    // going back to a previous version of XO not supporting CBT will create a full backup
+    // this will only do something after snapshot and transfer
+    if (this._exportedVm?.is_a_snapshot && this._settings.snapshotRetention === 0) {
+      const vdiRefs = await this._xapi.VM_getDisks(this._exportedVm?.$ref)
+      await xapi.call('VM.destroy', this._exportedVm.$ref)
+      for (const vdiRef of vdiRefs) {
+        try {
+          await xapi.VDI_dataDestroy(vdiRef)
```

Comment:
Shouldn't it be behind a flag?

I believe this should be disable by default for this month, but enabled on our test infra and we should explain in the PR (and blog post) how to enable.

And then, if all goes well, it should be turned on by default in the next release.